### ISSUE TITLE
Wrong positioning of sdtc:functionCode inside performer tag of CareTeamMemberAct.

### ISSUE THREAD

**Issue by SubhashKonar** [mdht/mdht#163]
[CCDA Validator](https://ett.healthit.gov/ett/#/validators/ccdar3#ccdaValdReport) if failing for the CCD with sdtc:functionCode tag inside performer tag.
![image](https://user-images.githubusercontent.com/64186900/117253806-3f9d1880-ae0d-11eb-88c8-8c26de5060f5.png)

Example for Care Teams XML  from [repo](https://github.com/HL7/C-CDA-Examples/tree/master/Guide%20Examples/Care%20Team%20Organizer_2.16.840.1.113883.10.20.22.4.500).
![image](https://user-images.githubusercontent.com/64186900/117254064-9276d000-ae0d-11eb-87da-43d27f3dc02a.png)

Care Teams Sample XML generated with MDHT library. 
![image](https://user-images.githubusercontent.com/64186900/117254235-c9e57c80-ae0d-11eb-9ed9-18c3818d94c6.png)

CCDA Validation is passing only when sdtc:functionCode is the first child of perfomer tag and fails if it is not the first child of performer tag.

The following CCD passes Validation.
![image](https://user-images.githubusercontent.com/64186900/117254462-0c0ebe00-ae0e-11eb-8be0-a28ca7e353c8.png)

The following CCD fails Validation.
![image](https://user-images.githubusercontent.com/64186900/117254235-c9e57c80-ae0d-11eb-9ed9-18c3818d94c6.png)

Can someone provide solution how to address the issue of positioning functionCode tag while using MDHT library ?

**Comment by swmuir** [comment]
It appears the corresponding ecore model used for generation is in a different order then the schema 

this accounts for the generation and the validation to be out of synch - will work on an update

**Comment by SubhashKonar** [comment]
@swmuir  Thanks!!! May I know by when can we expect the solution to this issue?

**Comment by swmuir** [comment]
here is after the update
<performer2 xmlns="urn:hl7-org:v3" xmlns:sdtc="urn:hl7-org:sdtc" nullFlavor="ASKU" typeCode="DIST">
  <realmCode/>
  <typeId nullFlavor="ASKU" assigningAuthorityName="Test" root="Test" extension="Test"/>
  <templateId/>
  <sdtc:functionCode/>
  <time/>
  <modeCode/>
  <assignedEntity nullFlavor="ASKU" classCode="ASSIGNED">
    <realmCode/>
    <typeId nullFlavor="ASKU" assigningAuthorityName="Test" root="Test" extension="Test"/>
    <templateId/>
    <id/>
    <code/>
    <addr/>
    <telecom/>
    <assignedPerson nullFlavor="ASKU" classCode="ANM" determinerCode="INSTANCE">
      <realmCode/>
      <typeId nullFlavor="ASKU" assigningAuthorityName="Test" root="Test" extension="Test"/>
      <templateId/>
      <name/>
      <sdtc:birthTime/>
    </assignedPerson>
    <representedOrganization nullFlavor="ASKU" classCode="NAT" determinerCode="INSTANCE">
      <realmCode/>
      <typeId nullFlavor="ASKU" assigningAuthorityName="Test" root="Test" extension="Test"/>
      <templateId/>
      <id/>
      <name/>
      <telecom/>
      <addr/>
      <standardIndustryClassCode/>
      <asOrganizationPartOf nullFlavor="ASKU" classCode="ACCESS">
        <realmCode/>
        <typeId nullFlavor="ASKU" assigningAuthorityName="Test" root="Test" extension="Test"/>
        <templateId/>
        <id/>
        <code/>
        <statusCode/>
        <effectiveTime/>
        <wholeOrganization nullFlavor="ASKU" classCode="NAT" determinerCode="INSTANCE">
          <realmCode/>
          <typeId/>
          <templateId/>
          <id/>
          <name/>
          <telecom/>
          <addr/>
          <standardIndustryClassCode/>
          <asOrganizationPartOf/>
        </wholeOrganization>
      </asOrganizationPartOf>
    </representedOrganization>
    <sdtc:patient>
      <sdtc:id/>
    </sdtc:patient>
  </assignedEntity>
</performer2>
will post soon

**Comment by SubhashKonar** [comment]
@swmuir I didn't get what you meant .Can you please elaborate? :) 
Updated code is not merged yet right?

**Comment by swmuir** [comment]
yes I need to push the update to the repository after testing which I did not get completed

**Comment by SubhashKonar** [comment]
Hi @swmuir Can you please push the update if testing is done. It will of great help if code is merged at the earliest.

**Comment by swmuir** [comment]
Yes sorry It was pushed I did not post back here

https://git.eclipse.org/r/c/mdht/org.eclipse.mdht/+/181314

How do you get the jars ?

**Comment by SubhashKonar** [comment]
can we update the github repo for mdht?

**Comment by SubhashKonar** [comment]
I download it from release

**Comment by SubhashKonar** [comment]
Can you please merge the code in this github repo as a release (https://github.com/mdht/mdht-models/releases) so that I can download the jar.

**Comment by swmuir** [comment]
The base code has moved here

https://git.eclipse.org/r/admin/repos/mdht/org.eclipse.mdht <https://git.eclipse.org/r/admin/repos/mdht/org.eclipse.mdht>

The MDHT is used to build a few components that we cannot push to eclipse 

We do have some maven support if that is better for you

**Comment by SubhashKonar** [comment]
Yes. Please provide the maven dependency.

**Comment by SubhashKonar** [comment]
Hi @swmuir  I tried searching in maven repository didn't find dependency xml.

**Comment by swmuir** [comment]
```
<distributionManagement>
                                <repository>
                                        <id>mdix</id>
                                        <name>7ed03140605b-releases</name>
                                        <url>http://artifactory.mdixinc.net:8082/artifactory/mdmiartifactory/libs-release</url>
                                </repository>
                                <snapshotRepository>
                                        <id>mdix</id>
                                        <name>7ed03140605b-snapshots</name>
                                        <url>http://artifactory.mdixinc.net:8081/artifactory/mdmiartifactory/libs-snapshot</url>
                                </snapshotRepository>
                        </distributionManagement>
```

**Comment by SubhashKonar** [comment]
Thanks @swmuir  , Can u please provide the version which need to be taken from this repo.

**Comment by SubhashKonar** [comment]
@swmuir I didn't find the updated jars in the distributionManagement provided above. Can you please guide me if this is the correct location/files which I'm looking into?

![image](https://user-images.githubusercontent.com/64186900/125085061-cc738500-e08f-11eb-9b73-7a2bdf526771.png)

**Comment by SubhashKonar** [comment]
> ```
> <distributionManagement>
>                              <repository>
>                                      <id>mdix</id>
>                                      <name>7ed03140605b-releases</name>
>                                      <url>http://artifactory.mdixinc.net:8082/artifactory/mdmiartifactory/libs-release</url>
>                              </repository>
>                              <snapshotRepository>
>                                      <id>mdix</id>
>                                      <name>7ed03140605b-snapshots</name>
>                                      <url>http://artifactory.mdixinc.net:8081/artifactory/mdmiartifactory/libs-snapshot</url>
>                              </snapshotRepository>
>                      </distributionManagement>
> ```
@swmuir I didn't find the updated jars in the distributionManagement provided above. Can you please guide me if this is the correct location/files which I'm looking into?

![image](https://user-images.githubusercontent.com/64186900/125085061-cc738500-e08f-11eb-9b73-7a2bdf526771.png)

**Comment by SubhashKonar** [comment]
Thanks for the updated jar,the issue is fixed.

Can you please let us know when it will be available in ( https://github.com/mdht/mdht-models/releases )
